src/Workspaces/CSharp/Portable/Extensions/SyntaxTriviaExtensions.cs

```diff
@@ -50,6 +50,11 @@ public static bool IsMultiLineComment(this SyntaxTrivia trivia)
             return trivia.Kind() == SyntaxKind.MultiLineCommentTrivia;
         }
 
+        public static bool IsShebangComment(this SyntaxTrivia trivia)
+        {
+            return trivia.Kind() == SyntaxKind.ShebangCommentTrivia;
```

Comment:
I don't see why it would need to be HashExclatationTrivia. 

SingleLineCommentTrivia is also not called SlashSlashTrivia